actions/src/main/java/com/redhat/service/bridge/actions/kafkatopic/KafkaTopicActionValidator.java

```diff
@@ -0,0 +1,19 @@
+package com.redhat.service.bridge.actions.kafkatopic;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import com.redhat.service.bridge.actions.ActionParameterValidator;
+import com.redhat.service.bridge.infra.models.actions.BaseAction;
+
+@ApplicationScoped
+public class KafkaTopicActionValidator implements ActionParameterValidator {
+
+    @Override
+    public boolean isValid(BaseAction baseAction) {
+        if (baseAction.getParameters() != null) {
+            String topic = baseAction.getParameters().get(KafkaTopicAction.TOPIC_PARAM);
+            return topic != null && !topic.isEmpty();
+        }
+        return false;
+    }
```

Comment:
@robpblake @r00ta 
Here is where I was thinking at the possibility to plug an [Hibernate Validator based validator](https://quarkus.io/guides/validation#json-mapping-and-validation) (and this can be different for each action)
It marshall/unmarshall the object so it is a bit unefficient but it is done only once during the saving of the action in the Manager so I think it is not an issue and we preserve module isolation/modularisation.
Wdyt?
(I have also suggested to return the list of violation instead of a boolean in the next comment so that we don't lose proper error reporting)
```suggestion
    @Inject
    Validator validator;
    
    @Inject
    ObjectMapper om;

    @Override
    public boolean isValid(BaseAction baseAction) {
        CustomActionBean bean = om.readValue(om.writeValueAsString(baseAction), CustomActionBean.class);
        Set<ConstraintViolation<Book>> violations = validator.validate(bean);
        return violations.isEmpty();
    }
    
    public static class CustomActionBean {
        @NotNull
        ....
    }
```

Reply:
Yep, works for me.